src/IBM.Watson.Assistant.v1/AssistantService.cs

```diff
@@ -81,10 +81,19 @@ public AssistantService(string version, string serviceName, IAuthenticator authe
         /// the previous response. (optional)</param>
         /// <param name="output">An output object that includes the response to the user, the dialog nodes that were
         /// triggered, and messages from the log. (optional)</param>
+        /// <param name="userId">A string value that identifies the user who is interacting with the workspace. The
+        /// client must provide a unique identifier for each individual end user who accesses the application. For
+        /// user-based plans, this user ID is used to identify unique users for billing purposes. This string cannot
+        /// contain carriage return, newline, or tab characters. If no value is specified in the input, **user_id** is
+        /// automatically set to the value of **context.conversation_id**.
+        ///
+        /// **Note:** This property is the same as the **user_id** property in the context metadata. If **user_id** is
+        /// specified in both locations in a message request, the value specified at the root is used.
+        /// (optional)</param>
         /// <param name="nodesVisitedDetails">Whether to include additional diagnostic information about the dialog
         /// nodes that were visited during processing of the message. (optional, default to false)</param>
         /// <returns><see cref="MessageResponse" />MessageResponse</returns>
-        public DetailedResponse<MessageResponse> Message(string workspaceId, MessageInput input = null, List<RuntimeIntent> intents = null, List<RuntimeEntity> entities = null, bool? alternateIntents = null, Context context = null, OutputData output = null, bool? nodesVisitedDetails = null)
+        public DetailedResponse<MessageResponse> Message(string workspaceId, MessageInput input = null, List<RuntimeIntent> intents = null, List<RuntimeEntity> entities = null, bool? alternateIntents = null, Context context = null, OutputData output = null, string userId = null, bool? nodesVisitedDetails = null)
```

Comment:
this new param should be moved to the end. prob need to see whats up with the generator/api def for this

Reply:
Oh Okay I'll create an issue to address it

Reply:
https://github.ibm.com/Watson/developer-experience/issues/9431